policy_tests/helper_fns.py

```diff
@@ -28,3 +28,7 @@ def doMkDir(dir):
         if e.errno != errno.EEXIST:
             raise    
 
+def is_64_bit_arch(arch):
```

Comment:
Seems redundant. `if '64' in arch` should do.

Reply:
This could go either way though. On one hand, we'll have to maintain a list of architectures here. On the other hand, having this function is more explicit.

Reply:
It's only used in xfail currently, so once the compiler issue is investigated and fixed this function can go away completely.